scripts/install_backend_python_libs.py

```diff
@@ -394,9 +394,21 @@ def _run_pip_command(cmd_parts):
         raise Exception('Error installing package')
 
 
+def pip_install_to_system(package, version):
+    """Installs third party libraries with pip to the current system.
```

Comment:
current system --> user's system

And then add a new paragraph:

Note: These libraries are installed to the user's default system-wide site-packages folder, not to a local Oppia third-party directory. (And then explain why we need to do this and that developers should not use this in general.)


(It is "site-packages", right?)